Bot/MessageHandlers/Utils/getIdFromMention.js

```diff
@@ -0,0 +1,30 @@
+export const GetIdFromMention = (mention, msg) => {
+	if (mention.startsWith("<@") && mention.endsWith(">")) {
```

Comment:
Prefer just exiting out instead of having all the code indented, E.G. `	if (!mention.startsWith("<@") || !mention.endsWith(">")) return;`